src/upload.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -e
+mc alias set remote "$STORAGE_ENDPOINT" "$STORAGE_ACCESS_KEY" "$STORAGE_SECRET_KEY" --api "$STORAGE_SIGNATURE_VERSION"
+mc cp --recursive /backups/output remote/"$STORAGE_BUCKET_NAME"
```

Comment:
```suggestion
mc cp --recursive /backups/output/ remote/"$STORAGE_BUCKET_NAME"
```

This way (with the trailing / for the source directory) it does not create a `output` folder in the bucket which mimics the current behaviour

Reply:
Oh, this is a nice find. Thanks. Will also update the test case then.